Taskfile.yaml

```diff
@@ -218,6 +238,19 @@ tasks:
       - echo " -  https://display-admin-client.local.itkdev.dk/admin"
       - echo " -  https://displayapiservice.local.itkdev.dk/client"
 
+  up:
```

Comment:
I can't see the situation where you want `up` as we don't really have a `stop`  - so, at the very least that should be added as well.

Reply:
Restarting docker and don't want all the data to be destroyed :)